### ISSUE TITLE
Bug: toHTML decodes html entities in triple-curlies

### ISSUE THREAD

**Issue by TehShrike** [ractivejs/ractive#2882]
## Description:

I'm using triple-curly braces to embed HTML into another template.

When that embedded HTML contains encoded HTML entities, they are incorrectly decoded in the `toHTML()` output string.

When output to the DOM, everything works fine.  The issue only happens with `toHTML()`.

## Versions affected:

Introduced in 0.5.8, still in 0.8.10

## Reproduction:

https://jsfiddle.net/qLz7j56t/2/

## Code:

```js
const r = window.r = new Ractive({
  el: '#main',
  template: `<p>{{{html}}}</p>`,
  data: {
    html: `<pre><code>This is &lt;strong&gt;legit&lt;/strong&gt;!</code></pre>`
  }
});


console.log(`Should be:
<p><pre><code>This is &lt;strong&gt;legit&lt;/strong&gt;!</code></pre></p>
`)
console.log(`Actually is:
${r.toHTML()}
`) // => <p><pre><code>This is <strong>legit</strong>!</code></pre></p>
```

**Comment by evs-chris** [comment]
It looks like there were some tests added to make sure triples worked in attributes, but there wasn't coverage for non-attribute triples. This is now fixed in edge and v0.8-dev. It'll go out with 0.8.11 when it gets cut. Thanks for the report!

**Comment by TehShrike** [comment]
Kickass turnaround time, thanks so much for the fix!
